include/libdnf/repo/repo.hpp

```diff
@@ -391,7 +391,11 @@ class Repo {
 
     void internalize();
 
-    void reset_metadata_expired();
+    /// If the repository is not already marked as expired, it checks for the presence of the storage cache expiration
+    /// attribute, and if the metadata_expire configuration value is set, also checks the modification times
+    /// of the master configuration file, the repository configuration file, and the cached primary file.
```

Comment:
`master` term is somehow cryptic but I understand that it can be `repo` file or `conf`. But I do not have a better term. May be something like `source`.

Reply:
I changed "master" to "main".